CHANGES.md

```diff
@@ -71,11 +71,15 @@
 
 ## New Features / Improvements
 
+* Improved batch performance of SparkRunner's GroupByKey ([#20943](https://github.com/apache/beam/pull/20943)).
+* Support OnWindowExpiration in Prism ([#32211](https://github.com/apache/beam/issues/32211)).
```

Comment:
Some of the changes listed here don't seem to be part of this PR

Reply:
Ah! My mistake. This happened while resolving git conflicts through the web UI.😅